helm/cluster-operator-resource-chart/templates/aws-cluster-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "core.giantswarm.io/v1alpha1"
+kind: AWSClusterConfig
```

Comment:
Did I undertand correctly that we do not need the manual `aws-cluster-config`, once this is implemented?
https://github.com/giantswarm/cluster-operator/blob/d0a9a11096b1a0a44e3ee2c24f1eccab0c06f7b3/service/controller/aws/v2/resource/awsconfig/create.go#L10

Reply:
I'm not sure I fully got what you mean, but your referenced `ApplyCreateChange` takes care of creating `AWSConfig` resources which is CR for `aws-operator` and contains provider specific cluster details. `AWSClusterConfig` CR is the one that drives `cluster-operator/service/controller/aws` framework and will be needed in the future. The thing is though that once I get Helm stuff sorted out and `cluster-service` & `kubernetesd` gets corresponding PRs merged, `cluster-service` will create/update/delete `AWSClusterConfig` when managing cluster e.g. with `gsctl`.

Reply:
Ah I got it all wrong. Thanks for sheding some light into the darkness. :fist_oncoming: